service/controller/v5/cloudconfig/cloud_config.go

```diff
@@ -242,43 +266,5 @@ func newCloudConfig(template string, params k8scloudconfig.Params) (string, erro
 		return "", microerror.Mask(err)
 	}
 
-	compressed, err := gzipBase64(cloudConfig.String())
-	if err != nil {
-		return "", microerror.Mask(err)
-	}
-
-	// cloud-config is compressed so we fit the tight 85kB limit of
-	// customData parameter.
-	//
-	// "Custom data in OSProfile must be in Base64 encoding and with
-	// a maximum length of 87380 characters."
-	//
-	//  87380 / 1024 = 85
-	customData := fmt.Sprintf(`#!/bin/bash
-D="/root/cloudinit"
-mkdir -m 700 -p ${D}
-echo -n "%s" | base64 -d | gzip -d -c > ${D}/cc
-coreos-cloudinit --from-file=${D}/cc`, compressed)
-
-	customData = base64.StdEncoding.EncodeToString([]byte(customData))
-	return customData, nil
-}
-
-func gzipBase64(s string) (string, error) {
-	buf := new(bytes.Buffer)
-
-	w, err := gzip.NewWriterLevel(buf, gzip.BestCompression)
-	if err != nil {
-		return "", microerror.Mask(err)
-	}
-	_, err = io.WriteString(w, s)
-	if err != nil {
-		return "", microerror.Mask(err)
-	}
-	err = w.Close()
-	if err != nil {
-		return "", microerror.Mask(err)
-	}
-
-	return base64.StdEncoding.EncodeToString(buf.Bytes()), nil
+	return cloudConfig.String(), nil
```

Comment:
This name is still confusing.

Reply:
It is the same across all the operators as the package name is still `k8scloudconfig`. Maybe we can change it after as kvm/aws are still using `cloudconfig`. Don't want to create this inconsistency right now.
 I can track that within issue about moving ignition into separate repo https://github.com/giantswarm/giantswarm/issues/5083

Reply:
👍